test/test_integration.py

```diff
@@ -333,25 +333,25 @@ def test_export_quantify_scheduler(self):
             self.assertEqual(
                 operations,
                 [
-                    "Rz(3.1415927, 'qreg[1]')",
-                    "Rxy(1.5707963, 1.5707963, 'qreg[1]')",
-                    "CZ (qreg[0], qreg[1])",
-                    "Rz(3.1415927, 'qreg[1]')",
-                    "Rxy(1.5707963, 1.5707963, 'qreg[1]')",
-                    "CZ (qreg[0], qreg[1])",
-                    "Rz(1.5707963, 'qreg[1]')",
-                    "Rxy(0.19634954, 1.5707963, 'qreg[1]')",
-                    "Rz(-1.5707963, 'qreg[1]')",
-                    "CZ (qreg[0], qreg[1])",
-                    "Rz(1.5707963, 'qreg[1]')",
-                    "Rxy(-0.19634954, 1.5707963, 'qreg[1]')",
-                    "Rz(-1.5707963, 'qreg[1]')",
-                    "CZ (qreg[0], qreg[1])",
-                    "Rz(0.19634954, 'qreg[0]')",
-                    "Rxy(-1.5707963, 1.5707963, 'qreg[0]')",
-                    "Rz(3.1415927, 'qreg[0]')",
-                    "Rz(3.1415927, 'qreg[1]')",
-                    "Rxy(1.5707963, 1.5707963, 'qreg[1]')",
+                    "Rz(-180, 'qreg[1]')",
```

Comment:
Oh sorry, I read `180` and thought that `pi` would be clearer. Especially `pi/16` for `11.25`. But that wouldn't be `pi`, you're right. Maybe `180/16` instead of `11.25` would be clearer? The rest of the angles are quite self descriptive.

Reply:
This is a test to check the generated output of the quantify scheduler transpiler. I don't see value in changing the transpiler such that it provides the output in fractions (also not sure how Quantify scheduler deals with fractions: they might be rounded); this will, in general, not be read by humans.